themes/default/content/docs/using-pulumi/pulumi-packages/troubleshoot-packages.md

```diff
@@ -0,0 +1,70 @@
+---
+title_tag: "How to Debug Pulumi Providers"
+meta_desc: "Learn the process for debugging Pulumi providers locally."
+title: Debug providers
+h1: Debug Pulumi providers
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    usingpulumi:
+        parent: pulumi-packages
+        identifier: how-to-debug-providers
+        weight: 2
+aliases:
+- /docs/guides/pulumi-packages/how-to-debug-providers/
+---
+
+When developing or troubleshooting Pulumi providers, you may need to debug the provider code locally. This guide walks you through starting your provider in debug mode, setting breakpoints, and running tests.
+
+### Starting the provider in debug mode
+
+### Example for GoLand
```

Comment:
The two examples are on the same level `###` than "Starting the provider"; should they be one lower?